src/pages/sources/SourcesListView.tsx

```diff
@@ -366,8 +468,207 @@ const SourcesListView: React.FunctionComponent = () => {
+            {connectionsData.unreachable.length ? connectionsData.unreachable.map((con) => (
+                <ListItem>{con.name}</ListItem>
+              )) : (<ListItem>N/A</ListItem>)}
+            </List>
+            <TextContent style={{margin: '1em 0'}}><h5><Icon status="success"><CheckCircleIcon /></Icon>  Successful connections</h5></TextContent>
+            <List isPlain isBordered>
+            {connectionsData.successful.length ? connectionsData.successful.map((con) => (
+                <ListItem>{con.name}</ListItem>
+              )) : (<ListItem>N/A</ListItem>)}
+            </List>
+          </Modal>
+      )}
+
+      {!!pendingDeleteSource && (
+            <Modal
+              variant={ModalVariant.small}
+              title="Permanently delete source"
+              isOpen={!!pendingDeleteSource}
+              onClose={() => setPendingDeleteSource(undefined)}
+              actions={[
+                <Button key="confirm" variant="danger" onClick={() => onDeleteSource(pendingDeleteSource)}>
+                  Delete
+                </Button>,
+                <Button key="cancel" variant="link" onClick={() => setPendingDeleteSource(undefined)}>
+                  Cancel
+                </Button>
+              ]}
+            >
+              Are you sure you want to delete the source "{pendingDeleteSource.name}"
+            </Modal>
+      )}
+
+      {addSourceModal && (
+        <AddSourceModal
+          type={addSourceModal}
+          onClose={() => setAddSourceModal(undefined)}
+          onSubmit={onAddSource}
+        />
+        // <Modal
+        //   variant={ModalVariant.medium}
+        //   onClose={() => setAddSourceModal(undefined)}
+        //   isOpen={!!addSourceModal}
+        //   title={`Add sorce: ${addSourceModal}`}
+        // >
+        //   {addSourceModal}
+        // </Modal>
+      )}
+
+      {scanSelected && (
+        <SourcesScanModal 
+          onClose={() => setScanSelected(undefined)}
+          onSubmit={onRunScan}
+          sources={scanSelected} />
+      )}
+      <AlertGroup isToast isLiveRegion>
+        {alerts.map(({ key, variant, title }) => (
+          <Alert
+            timeout={8000}
+            onTimeout={() => key && removeAlert(key)}
+            variant={AlertVariant[variant || 'info']}
+            title={title}
+            actionClose={
+              <AlertActionCloseButton
+                title={title as string}
+                variantLabel={`${variant} alert`}
+                onClose={() => key && removeAlert(key)}
+            />
+          }
+          key={key}
+        />
+        ))}
+      </AlertGroup>
     </PageSection>
   );
 };
 
 export default SourcesListView;
+
+
+
+  // const sourcesData = {
+  //   "count": 2,
+  //   "next": null,
+  //   "previous": null,
+  //   "results": [
+  //       {
+  //           "id": 1,
+  //           "name": "Peripherals",
+  //           "source_type": "openshift",
```

Comment:
This payload is being used for manual testing, right? I suggest removing it from here and saving it in a folder with example payloads as json files.